larpandora/LArPandoraInterface/LArPandoraGeometry.cxx

```diff
@@ -406,24 +404,23 @@ namespace lar_pandora {
           if (dThetaU > maxDeltaTheta || dThetaV > maxDeltaTheta || dThetaW > maxDeltaTheta)
             continue;
 
-          double localCoord2[3] = {0., 0., 0.};
-          double worldCoord2[3] = {0., 0., 0.};
-          theTpc2.LocalToWorld(localCoord2, worldCoord2);
+	  double dminx2 = theTpc2.ActiveBoundingBox().MinX();
+	  double dmaxx2 = theTpc2.ActiveBoundingBox().MaxX();
```

Comment:
I think I would be inclined to just bring forward the initialisation of the driftMinX2 and driftMaxX2 from below to avoid the need to assign these quantities twice.